server/cli/launch.py

```diff
@@ -13,6 +14,10 @@
 from server.utils.constants import MODES
 
 
+# anything bigger than this will generate a special message
+BIG_FILE_SIZE_THRESHOLD = 100 * 2**20
```

Comment:
Just so I don't have to do any real thinking...

```suggestion
BIG_FILE_SIZE_THRESHOLD = 100 * 2**20 #100MB
```

Reply:
will fix